src/django/api/models.py

```diff
@@ -2490,6 +2492,36 @@ def post_save(sender, **kwargs):
             index_extendedfields([instance.facility.id])
 
 
+class Event(models.Model):
+    EVENT_TYPE_CHOICES = (
+        ('CREATE', 'CREATE'),
+        ('UPDATE', 'UPDATE'),
+        ('DELETE', 'DELETE'),
+        ('MERGE', 'MERGE'),
+        ('SPLIT', 'SPLIT'),
+        ('MOVE', 'MOVE'),
+        ('OTHER', 'OTHER'),
+        ('ASSOCIATE', 'ASSOCIATE'),
+        ('DISSOCIATE', 'DISSOCIATE'),
+        ('CLAIM', 'CLAIM'),
+        ('CLAIM_UPDATE', 'CLAIM_UPDATE'),
+        ('CLAIM_REVOKE', 'CLAIM_REVOKE'),
+    )
+
+    content_type = models.ForeignKey(ContentType, on_delete=models.CASCADE)
+    object_id = models.CharField(max_length=32)
+    content_object = GenericForeignKey('content_type', 'object_id')
+    event_type = models.CharField(max_length=50, choices=EVENT_TYPE_CHOICES)
+    event_time = models.DateTimeField()
+    event_details = models.JSONField()
```

Comment:
Most other models we specify `help_text` - is there a reason to skip it for this one?

Reply:
Help text added - thanks!